src/components/explorer/ExplorerIndividualContent.js

```diff
@@ -68,33 +76,71 @@ const ExplorerIndividualContent = () => {
+    useEffect(() => {
+        // Set individual resources in the store for OntologyTerm rendering with no prop drilling
+        if (resources) {
+            dispatch(setIndividualResourcesTuple(resources));
+        }
+    }, [dispatch, resources]);
 
     const overviewUrl = `${individualUrl}/overview`;
     const phenotypicFeaturesUrl = `${individualUrl}/phenotypic-features`;
     const biosamplesUrl = `${individualUrl}/biosamples`;
     const experimentsUrl = `${individualUrl}/experiments`;
-    const variantsUrl = `${individualUrl}/variants`;
-    const genesUrl = `${individualUrl}/genes`;
     const diseasesUrl = `${individualUrl}/diseases`;
     const ontologiesUrl = `${individualUrl}/ontologies`;
     const tracksUrl = `${individualUrl}/tracks`;
     const phenopacketsUrl = `${individualUrl}/phenopackets`;
+    const interpretationsUrl = `${individualUrl}/interpretations`;
+    const medicalActionsUrl = `${individualUrl}/medical-actions`;
+    const measurementsUrl = `${individualUrl}/measurements`;
 
+    const individualPhenopackets = individual?.phenopackets ?? [];
     const individualMenu = [
         {url: overviewUrl, style: {marginLeft: "4px"}, text: "Overview"},
-        {url: phenotypicFeaturesUrl, text: "Phenotypic Features"},
-        {url: biosamplesUrl, text: "Biosamples"},
-        {url: experimentsUrl, text: "Experiments"},
+        {
+            url: phenotypicFeaturesUrl,
+            text: "Phenotypic Features",
+            disabled: useIsDataEmpty(individualPhenopackets, "phenotypic_features"),
+        },
+        {
+            url: biosamplesUrl,
+            text: "Biosamples",
+            disabled: useIsDataEmpty(individualPhenopackets, "biosamples"),
+        },
+        {
+            url: experimentsUrl,
+            text: "Experiments",
+            disabled: useIsDataEmpty(
+                useDeduplicatedIndividualBiosamples(individual),
+                "experiments",
+            ),
+        },
+        {
+            url: interpretationsUrl,
+            text: "Interpretations",
+            disabled: useIsDataEmpty(individualPhenopackets, "interpretations"),
+        },
         {url: tracksUrl, text: "Tracks"},
-        {url: variantsUrl, text: "Variants"},
-        {url: genesUrl, text: "Genes"},
-        {url: diseasesUrl, text: "Diseases"},
+        {
+            url: diseasesUrl,
+            text: "Diseases",
+            disabled: useIsDataEmpty(individualPhenopackets, "diseases"),
+        },
         {url: ontologiesUrl, text: "Ontologies"},
+        {
+            url: medicalActionsUrl,
+            text: "Medical Actions",
+            disabled: useIsDataEmpty(individualPhenopackets, "medical_actions"),
+        },
+        {
+            url: measurementsUrl,
+            text: "Measurements",
+            disabled: useIsDataEmpty(individualPhenopackets, "measurements"),
+        },
         {url: phenopacketsUrl, text: "Phenopackets JSON"},
```

Comment:
reorder the menu - basically in order of what's interesting to researchers + grouped together in terms of data type (loosely). tentatively, I'm thinking:

* overview

Biosample block:
* biosamples
* measurements
* phenotypic features
* diseases
* interpretations
* medical actions

Experiment block:
* experiments
* tracks

Extra:
* ontologies
* phenopackets

routes should be re-ordered too to match.

Reply:
Reordered the menus/routes. 
By "grouping together" do you mean adding sub-menus?